### PR TITLE
email notification edits

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/project-access.md [hunk 1]
```diff
@@ -42,7 +42,12 @@ You can also view the Access level from your Projects page.<br/><img src={useBas
 
 ## Email Notifications
 
-Email notifications will alert you when a test starts failing, and notify you again when the test is back in full working order. The incident identifier number, used to track the events, will be the same as the test ID.
+Email notifications will alert you when a test starts failing, then further consequent failures will be muted for 30 minutes. A new notification will be sent when the test is back in full working order or the first failing execution after the muted period.
+
+:::note IMPORTANT
+The email notification mechanism we have in place consist in an email notification when the test starts failing (the subject of the email will contain "Incident started"). A new email notification will be sent after 30 minutes if the test is still failing (the subject of the email will contain "Incident in progress"), this notification is sent only if your tests runs less then every 30 minutes. As soon as the test execution passes, you will be notified with an email notification (the subject of the email will contain "Incident resolved")
+:::
```
Comment (kimsauce) [inline docs/api-testing/project-access.md:49]: Some repetition here. I suggest consolidating, @lauramthomas.

**Review by kimsauce — DISMISSED** [review]
Left a recommendation

**Review by lauramthomas — CHANGES_REQUESTED** [review]
@LoryApiFortress I added my own edits to your review.

**Review by lysannep — APPROVED** [review]
Looks good!

**Pull request merged**